eif_utils/src/lib.rs

```diff
@@ -257,6 +237,10 @@ impl<T: Digest + Debug + Write + Clone> EifBuilder<T> {
 
         if self.sign_info.is_some() {
             result[DEFAULT_SECTIONS_COUNT as usize + self.ramdisks.len()] = self.signature_offset();
+            result[DEFAULT_SECTIONS_COUNT as usize + self.ramdisks.len() + 1] =
```

Comment:
Should add a comment explaining this line.

Reply:
Removed this piece in the last commit: metadata section was put higher (after cmdline and before ram disks sections)